<?xml version="1.0"?>
<launch>
  <arg name="image" default="/usb_cam/image_raw" />
  <arg name="model_name" default="deeplabv3" />
  <arg name="version" default="mobilenet_v3_large" />

  <node pkg="pytorch_ros" type="semantic_segmentation.py" name="semantic_segmentation" output="screen">
    <remap from="image" to="$(arg image)" />
    <param name="model_name" value="$(arg model_name)" />
    <param name="version" type="str" value="$(arg version)" />
  </node>
</launch>